kmk/extensions/layers.py

```diff
@@ -0,0 +1,164 @@
+import kmk.handlers.modtap as modtap
+from kmk.extensions import Extension
+from kmk.key_validators import layer_key_validator, mod_tap_validator
+from kmk.keys import make_argumented_key
+from kmk.kmktime import ticks_diff, ticks_ms
+
+
+class Layers(Extension):
```

Comment:
this whole class is so cool! I feel like I understand how layers are implemented here